region/lowernorfair/east.json

```diff
@@ -1397,7 +1397,11 @@
                           "count": 1
                         }]}
                       ]},
-                      "SpringBall"
+                      "SpringBall",
+                      {"and":[
+                        "canXRayStandUp",
+                        "canCeilingClip"
```

Comment:
How does this even look and does it really not add any heatframes compared to Spring Ball?

Reply:
For future archeology https://www.youtube.com/watch?v=qHOT9zY0w2Q